### PR TITLE
Add new operations groups to Pioneer

### PR DESCRIPTION
**Pull request by DzhideX**
This PR aims to implement this issue: https://github.com/Joystream/joystream/issues/2629

Changes:
- Add WGs Alpha, Beta, Gamma to Working Groups tab. I've just copied the wording from the previous Operations group card. It may be a good idea to change this now?
- Add new WGs to proposals.
- Add new WGs to the Tokenomics tab. The Spending and Stake distribution table and charts are getting crowded but I think it shouldn't really pose a problem seeing as Pioneer v2 is almost around the corner.
- Update types.

Edit:
These issues have also been added to this PR: https://github.com/Joystream/joystream/issues/2627, https://github.com/Joystream/joystream/issues/2628

### CODE REVIEW

File: pioneer/packages/joy-proposals/src/forms/GenericWorkingGroupProposalForm.tsx [hunk 1]
```diff
@@ -45,7 +45,17 @@ export type FormInnerProps = ProposalFormInnerProps<FormContainerProps, FormValu
 
 const availableGroupsOptions = Object.keys(WorkingGroupDef)
   .filter((wgKey) => wgKey !== 'Gateway') // Gateway group not yet supported!
-  .map((wgKey) => ({ text: wgKey + ' Working Group', value: wgKey }));
+  .map((wgKey) => {
+    let text = wgKey + 'Working Group';
```
Comment (Lezek123) [inline pioneer/packages/joy-proposals/src/forms/GenericWorkingGroupProposalForm.tsx]: I think you missed the spacebar:
```suggestion
    let text = `${wgKey} Working Group`;
```

**Review by Lezek123 — COMMENTED** [review]
Looks good, just 2 additional observations:
- Runtime type name has been changed from `DAOId` to `DaoId` which is not reflected in `@joystream/types`
- The tokenomics module requires by far the most manual work when updating working groups compared to any other affected module. Since all working groups have exactly the same logic, will it be possible to refactor the tokenomics module in such a way that the overhead of updating it is minimal? First thing that comes to mind is the multiplication of `SpendingAndStakeTableRow`.

**Comment by DzhideX** [comment]
I've gone through and implemented your comments. 

One thing to note wrt `DaoId`, there are still some remaining `DAOId`'s in the code, namely:
- There's a comment in `/runtime-modules/content/src/tests/mock.rs` .
- A bunch of references in `/runtime-modules/common/src/storage.rs` and `/query-node/generated/types/typedefs.json`.

I think the rust files shouldn't be a problem but what about the `typedefs.json`?

File: pioneer/packages/joy-tokenomics/src/Overview/SpendingAndStakeDistributionTable.tsx [hunk 2]
```diff
@@ -168,150 +159,59 @@ const SpendingAndStakeDistributionTable: React.FC<{data?: TokenomicsData; status
...
+                groupEarningDollar={displayStatusData(groupType, 'rewardsPerWeek')}
+                earningShare={data && `${round(data[groupType].rewardsShare * 100)}`}
+                groupStake={data && `${data[groupType].totalStake}`}
+                groupStakeDollar={displayStatusData(groupType, 'totalStake')}
+                stakeShare={data && `${round(data[groupType].stakeShare * 100)}`}
+                color={color}
+              />
+              <SpendingAndStakeTableRow
+                role={width <= lead.titleCutoff ? lead.shortTitle : lead.title}
+                helpContent={helpText}
```
Comment (Lezek123) [inline pioneer/packages/joy-tokenomics/src/Overview/SpendingAndStakeDistributionTable.tsx]:
```suggestion
                helpContent={lead.helpText}
```

**Review by Lezek123 — CHANGES_REQUESTED** [review]

**Comment by DzhideX** [comment]
Latest changes:
- Implement distribution group in Tokenomics, Working groups section and Proposals.
- Fix lead help text in `SpendingAndStakeDistributionTable`.
- Rename `contentDirectoryWorkingGroup` to `contentWorkingGroup` where necessary.

**Review by Lezek123 — APPROVED** [review]
Fantastic work, thank you!

**Pull request merged**